universal-login-react/src/core/utils/getSuggestionType.ts

```diff
@@ -23,5 +26,5 @@ const isSingleCreation = (creations: string[], connections: string[]) =>
 const isSingleConnection = (creations: string[], connections: string[], actions: WalletSuggestionAction[]) =>
   !actions.includes(WalletSuggestionAction.recover) && creations.length === 0 && connections.length === 1;
 
-const isKeepTyping = (creations: string[], connections: string[]) =>
-  creations.length === 0 && connections.length === 0;
+const isNone = (creations: string[], connections: string[], source: string) =>
```

Comment:
I don't like`isNone`, maybe `isNoSuggestion` or something like that?

Reply:
@cehali @rzadp ?

Reply:
How about blank?
`SuggestionType = 'Blank'`, `const isBlank = ...`

Reply:
`Blank` sounds like there are suggestions but they are empty/plain, if that makes sense. I like the original name of `None`

Reply:
`NoSuggestions` / `isNoSuggestions`

Reply:
Simplify and inline, see above #massivenamediscussion